www/front_src/src/Header/PollerMenu/index.tsx

```diff
@@ -0,0 +1,254 @@
+import * as React from 'react';
+
+import { useTranslation } from 'react-i18next';
+import { isEmpty, isNil } from 'ramda';
+import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+import classnames from 'classnames';
+import clsx from 'clsx';
+import { useAtomValue } from 'jotai/utils';
+import { useHistory } from 'react-router';
+
+import PollerIcon from '@material-ui/icons/DeviceHub';
+import {
+  Button,
+  ClickAwayListener,
+  makeStyles,
+  Paper,
+  Typography,
+} from '@material-ui/core';
+
+import {
+  getData,
+  useRequest,
+  IconHeader,
+  SubmenuHeader,
+  IconToggleSubmenu,
+} from '@centreon/ui';
+import { refreshIntervalAtom } from '@centreon/ui-context';
+
+import styles from '../header.scss';
+import { allowedPagesSelector } from '../../redux/selectors/navigation/allowedPages';
+import MenuLoader from '../../components/MenuLoader';
+
+import {
+  labelAllPollers,
+  labelConfigurePollers,
+  labelDatabaseUpdatesNotActive,
+  labelLatencyDetected,
+  labelPoller,
+  labelPollerNotRunning,
+} from './translatedLabels';
+import ExportConfiguration from './ExportConfiguration';
+import PollerStatusIcon from './PollerStatusIcon';
+
+export const pollerConfigurationNumberPage = '60901';
+
+const pollerIssueKeyToMessage = {
+  database: labelDatabaseUpdatesNotActive,
+  latency: labelLatencyDetected,
+  stability: labelPollerNotRunning,
+};
+
+export interface Issue {
+  critical: number;
+  total: number;
+  warning: number;
+}
```

Comment:
As they are used in several files, you can extract `Issue` and `Issues` interfaces to a specific `models` file